Fedoraware/Fedoraware-TF2/src/SDK/Main/BaseEntity/BaseEntity.h

```diff
@@ -701,6 +701,26 @@ class CBaseEntity
 		}
 	}
 
+	__inline bool IsCredits()
+	{
+		switch (GetClassID())
```

Comment:
You could simply do: `return GetClassID() == ETFClassID::CCurrencyPack;` instead.